src/docs/maintaining-the-schema.md

```diff
@@ -181,34 +204,8 @@ The text that comes immediately after the `nmdc` prefix and colon, and before th
 Typecodes must correspond 1:1 to a class in the NMDC schema. The typecodes currently in use are available from the
 nmdc-runtime API: https://api.microbiomedata.org/nmdcschema/typecodes
 
-The typecodes for each class are auto-discovered each time the nmdc-runtime API is restarted, 
-and the API is restarted each time it is bound to a new version of the nmdc-schem.
-
-We do not currently have any criteria for good typecodes, other than they are ideally between 3 and 6 characters in length.
-
-## unresolved
```

Comment:
@turbomam - this is the place in the diff where I pulled the "unresolved list and made it into this ticket: https://github.com/microbiomedata/nmdc-schema/issues/2318